docs/programming/ace_in_a_nutshell.md

```diff
@@ -159,6 +163,19 @@ Instead of writing `main()` function you just `#include` this file and define:
 - `genericLoop()` - called in a loop until game gets closed
 - `genericDestroy()` - for freeing previously created managers
 
+If you preffer exiting your game in other ways than by calling `gameExit`, you
+can tune game main loop condition by defining `GENERIC_MAIN_LOOP_CONDITION`
```

Comment:
And what is generic main.h? It's not mentioned until this point. This chapter is about basic game state concept, not about generic main.

Maybe move it to tutorial or describe it in generic.h as comment?

Reply:
This chapter as a whole literally explains everything about `main.h`... So... Where is the problem?

Reply:
sorry, I was basing this on my memory and diffs, haven't looked at the whole text. You're right, it's good.